src/tests/contract_test/utils.cljs

```diff
@@ -1,14 +1,14 @@
 (ns tests.contract-test.utils
   (:require
+    [promesa.core :as p]
     [status-im.common.json-rpc.events :as rpc-events]
     [utils.number]))
 
-(defn call-rpc-endpoint
-  [{:keys [rpc-endpoint
-           params
-           action
-           on-error]}]
-  (rpc-events/call {:method     rpc-endpoint
-                    :params     params
-                    :on-success action
-                    :on-error   on-error}))
+(defn call-rpc
+  [method & args]
```

Comment:
I could change the signature to become very similar to a synchronous function call because now there's no point in passing `on-success` and `on-error` callbacks.